### PR TITLE
Workspaces - navigation and header

### PR DESCRIPTION
**Pull request by radekkaluzik**
### Description
This PR implements first part of RHCLOUD-31947 - sets up routing and adds header for Workspaces. TableView will follow.

[RHCLOUD-31947](https://issues.redhat.com/browse/RHCLOUD-31947)

---

### Screenshots
image

---

### Checklist ☑️
- [x] PR only fixes one issue or story 
- [x] Change reviewed for extraneous code 
- [x] UI best practices adhered to 
- [x] Commits squashed and meaningfully named 
- [x] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [ ] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

File: src/smart-components/workspaces/workspaces.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <ContentHeader
+        title={intl.formatMessage(messages.workspaces)}
+        subtitle={intl.formatMessage(messages.workspacesSubtitle)}
+        // TODO: clarify with UX whether or not an icon will be used
+        // icon={<img src="/apps/frontend-assets/rbac-landing/rbac-landing-icon.svg" className="rbac-overview-icon" alt="RBAC landing page icon" />}
+        linkProps={{
+          label: intl.formatMessage(messages.workspacesLearnMore),
+          isExternal: true,
+          // TODO: how to pass URL to linkProps?
+          //to: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/user_access_configuration_guide_for_role-based_access_control_rbac/index',
```
Comment (fhlavac) [inline src/smart-components/workspaces/workspaces.tsx:21]:
```suggestion
          href: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/user_access_configuration_guide_for_role-based_access_control_rbac/index',
```
linkProps extend PF ButtonProps, so `href` can be used

**Comment by fhlavac** [comment]
@radekkaluzik apart from the link and icon clarification it looks good! Could you please also add a cypress test to check that the new content is visible?

File: src/smart-components/workspaces/workspaces.tsx (hunk 1 above)
Comment (radekkaluzik) [reply src/smart-components/workspaces/workspaces.tsx]: Fixed, thank you

**Comment by radekkaluzik** [comment]
@fhlavac It was confirmed by UX that icon is not required, URL will be provided later. Added Cypress test.

**Pull request merged**